plasmapy/utils/checks.py

```diff
@@ -316,6 +257,73 @@ def _check_quantity(arg, argname, funcname, units, can_be_negative=True,
     elif not can_be_inf and np.any(np.isinf(arg.value)):
         raise ValueError(f"{valueerror_message} infs.")
 
+    return arg
+
+
+def check_relativistic(func=None, betafrac=0.1):
+    r"""
+    Warns or raises an exception when the output of the decorated
+    function is greater than `betafrac` times the speed of light.
+
+    Parameters
+    ----------
+    func : `function`, optional
+        The function to decorate.
+
+    betafrac : float, optional
+        The minimum fraction of the speed of light that will raise a
+        `~plasmapy.utils.RelativityWarning`. Defaults to 0.1.
```

Comment:
Same question here. Perhaps 0.01 would be more sensible?

Reply:
Sure. Will change.